suite-native/discovery/src/discoveryMiddleware.ts

```diff
@@ -55,6 +65,29 @@ export const prepareDiscoveryMiddleware = createMiddlewareWithExtraDeps(
             );
         }
 
+        // this is for further continual rerunning discovery for various other reasons
+        if (
+            selectDeviceThunk.fulfilled.match(action) ||
+            setEnabledDiscoveryNetworkSymbols.match(action) ||
+            toggleEnabledDiscoveryNetworkSymbols.match(action) ||
+            accountsActions.changeAccountVisibility.match(action) ||
```

Comment:
Not sure about this one, because changeAccountVisibility is being dispatched only while adding new account and before `TrezorConnect.getAccountInfo` call. Maybe it's better to check for `addAccountByDescriptorThunk.fullfilled`?

Reply:
b675fc3e18